src/components/BuildConfigCreateEditPage/BuildConfigCreateEditPage.tsx

```diff
@@ -253,14 +258,62 @@ export const BuildConfigCreateEditPage = ({ isEditPage = false }: IBuildConfigCr
   };
 
   const submitEdit = (data: IFieldValues) => {
-    return Promise.resolve();
+    const buildConfig = {
+      name: data.name,
+      description: data.description,
+      environment: selectedEnvironment,
+      buildType: data.buildType,
+      buildScript: data.buildScript,
+      brewPullActive: data.brewPullActive,
+      scmRepository: selectedScmRepository,
+      scmRevision: data.scmRevision,
+      productVersion: selectedProductVersion,
+      parameters: Object.fromEntries(Object.entries(buildParamData).map(([k, v]) => [k, v.value])),
+    } as BuildConfiguration;
+
+    const patchData = createSafePatch(serviceContainerEditPageGet.data!, buildConfig);
```

Comment:
There is high amount of unchanged parameters when only Name is changed and submitted.

Reply:
Only nested entities' ids are now compared.